packages/cli/deploy/docker/client-build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# Build script for CLI image executed from Dockerfile
+echo Getting latest container image updates...
+apt-get update && apt-get upgrade -y > /var/log/build.log
+
+# The following two steps will be removed from the final build script:
+apt-get install mlocate -y >> /var/log/build.log
```

Comment:
Is this a TODO?

Reply:
Yes it is - once I’m happy that the image provides for consistently robust containers, this can come out to reduce the image size.